libs/langchain/langchain/document_loaders/web_base.py

```diff
@@ -63,6 +63,8 @@ def __init__(
         verify_ssl: Optional[bool] = True,
         proxies: Optional[dict] = None,
         continue_on_failure: Optional[bool] = False,
+        autoset_encoding: Optional[bool] = True,
```

Comment:
I think arguably the default should be `False` but I left it as `True` so as to avoid changing the behavior out from under anyone who came to rely on this behavior in the last 3 months.